### PR TITLE
Minor Intro Doc Improvements

### PR DESCRIPTION
**Pull request by danehans**
Makes minor improvements to the introduction document.

/assign @robscott 
/cc @mark-church

### CODE REVIEW

File: docs-src/index.md [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
-How do the Service APIs improve upon current standards like Ingress?
+How do Service APIs improve upon current standards like Ingress?
 
 - **More expressive** - They express more core functionality for things like
 header-based matching, traffic weighting, and other capabilities that were only
-possible through custom means in the Ingress spec.    
+possible in Ingress through custom means.    
 - **More extensible** - They allow for custom resources to be linked at multiple
-layers within its resources. This allows for more granular customization at the 
+layers of the APIs. This allows for more granular customization at the
```
Comment (hbagdi) [inline docs-src/index.md:27]: Suggestion: They allow for custom resources to be linked at ~multiple~ **various** layers of the ~APIs~**API**?

**Comment by robscott** [comment]
Thanks!

/approve
/hold for Harry's suggestion

**Comment by danehans** [comment]
Commit `24d5ea6` resolves https://github.com/kubernetes-sigs/service-apis/pull/468#discussion_r526343453.

/hold cancel

**Comment by robscott** [comment]
Thanks!

/lgtm

**Pull request merged**